bokehjs/src/coffee/models/scales/linear_scale.coffee

```diff
@@ -0,0 +1,60 @@
+import {Scale} from "./scale"
+
+import * as p from "core/properties"
+
+export class LinearScale extends Scale
+
+  initialize: (attrs, options) ->
+    super(attrs, options)
+
+    @define_computed_property('state', @_state, true)
+    @add_dependencies('state', this, ['source_range', 'target_range'])
+    @add_dependencies('state', @source_range, ['start', 'end'])
+    @add_dependencies('state', @target_range, ['start', 'end'])
```

Comment:
I'm happy to do that, but not in this PR which is more than large enough already. There will be more mapper PRs shortly that are smaller and would be better for stashing this kind of incidental cleanup.